src/modules/unit/components/MapView.js

```diff
@@ -76,17 +76,16 @@ class MapView extends Component {
   }
 
   centerMapToUnit = (unit: Object) => {
-    if (this.state.isMobile) {
-      const location = getUnitPosition(unit);
-      location[0] += 0.035;
-      location[1] -= 0.005;
-      // For some reason could not use reverse here so had to do this weird way.
-      this.refs.map.leafletElement.setView(location, DEFAULT_ZOOM);
+    const location = getUnitPosition(unit);
+    if (!this.state.isMobile) {
```

Comment:
Here the logic is flipped unnecessarily. I.e., previously the mobile logic was handled first and some other things have been changed as well. I understand why this has happened by looking at your earlier commit. Generally speaking, it's usually for the best to try an attempt to make changes which address the problem exactly.

However, the refactoring you have done seems to improve code quality so I am happy to let the changes tag along.

Reply:
Oh, I see. The logic changed anyway a bit on desktop as I didn't see any reason why to adjust the center point on desktop.